R/pkg/NAMESPACE

```diff
@@ -6,10 +6,15 @@ importFrom(methods, setGeneric, setMethod, setOldClass)
 #useDynLib(SparkR, stringHashCode)
 
 # S3 methods exported
+export("sparkR.session.getOrCreate")
```

Comment:
While the naming is accurate, this seems more verbose than before. So one option that i've been thinking about is what if we broke some backward compatibility and returned a SparkSession object from `sparkR.init` 
The other option is to just call it `sparkR.session` and `sparkR.session.stop`

@rxin Any thoughts on this ?